fix(sdk): manual fixes for module level ttag calls causing broken translations

npretto opened the pull request:
### Description

Part of EMB-335, manually change some `t` calls to make them work in the context of the sdk and static/public, where the locale is not loaded when modules initially loads,

Changes:
- changed labels defined in `frontend/src/metabase/parameters/components/UpdateFilterButton/getUpdateButtonProps.ts`
- `uiName` -> `getUiName()` on visualizations (for @metabase/core-frontend-dashviz )

### How to verify

For the viz names the easier way is to start sdk storybook (`yarn storybook-embedding-sdk`, [full instructions and setup here](https://github.com/metabase/metabase/blob/daab776f89b1d01519cf1eaf8e5069e7f63f1175/enterprise/frontend/src/embedding-sdk/dev.md?plain=1#L23-L41)), and go to http://localhost:6006/?path=/story/embeddingsdk-interactivequestion--default&globals=locale:de (notice the locale in the flags)

Note: for the

### Demo

<img width="805" alt="image" src="https://github.com/user-attachments/assets/0c2d0f50-08ec-49c8-a930-c17ada59744c" />

npretto committed:
fix: filter button labels

npretto committed:
fix: part of EMB-335, uiName: t => getUiName: () => t

alxnddr reviewed (commented):
Looks good! Can you please have a look at the `frontend/src/metabase/visualizations/components/Visualization/Visualization.unit.spec.js` spec which failed?

npretto committed:
fix linting issue

npretto committed:
fixed unit test mock missing getUiName
frontend/src/metabase/visualizations/components/Visualization/Visualization.unit.spec.js
missing

npretto committed:
make application-permissions.cy.spec.js pass

heypoom reviewed (approved):
All `getUiName` changes looks correct to me, thanks!
